packages/hardhat/hardhat.config.js

```diff
@@ -95,7 +95,8 @@ module.exports = {
       },
     },
     goerli: {
-      url: "https://goerli.infura.io/v3/460f40a260564ac4a4f4b3fffb032dad", // <---- YOUR INFURA ID! (or it won't work)
+      // TODO: Remove dard-coded INFURA ID
+      url: "https://goerli.infura.io/v3/XXXXXXXXXXXXXXXXXXXXXXXXX", // <---- YOUR INFURA ID! (or it won't work)
```

Comment:
the XXXX desired?

Reply:
This is code from the scaffold-eth template with some made up API. Will eventually pull all these from .env. Didn't want out=r actual infura key in code - hence the XXXX...
If you need it to work locally you will need to replace with a working infura key